components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1162,18 +1194,18 @@ private Condition createConditionForSingleAttributeFilter(String domainName, Nod
      * @throws CharonException
      */
     private List<Object> getMultiAttributeFilteredUsers(Node node, Map<String, Boolean> requiredAttributes,
-                                                        int offset, int limit, String sortBy, String sortOrder,
+                                                        int offset, Integer limit, String sortBy, String sortOrder,
```

Comment:
Is it necessary to change the signature

Reply:
fixed: 8f8c20292c4fac76778a8aa95490675023d7d063